core/string/ustring.cpp

```diff
@@ -1135,6 +1135,12 @@ String String::get_slice(String p_splitter, int p_slice) const {
 	if (p_slice < 0) {
 		return "";
 	}
+
+    // Shortcut / hack for splitting by empty strings
+    if (p_splitter.is_empty()) {
+        return substr(p_slice, 1);
```

Comment:
`return p_slice < length() ? *this[p_slice] : ""`

Also add punctuation in the comment as per our style guide.

Reply:
I was not able to find anywhere where the empty string behavior was used. I have also added a period to the comment.

Reply:
(oof I C++'d all over myself)